lib/jsdom/living/nodes/Node-impl.js

```diff
@@ -93,7 +117,7 @@ function isHostInclusiveAncestor(nodeImplA, nodeImplB) {
   return false;
 }
 
-class NodeImpl extends EventTargetImpl {
+exports.implementation = class NodeImpl extends EventTargetImpl {
```

Comment:
Revert this.

```suggestion
class NodeImpl extends EventTargetImpl {
```

Reply:
Well, 57&nbsp;instances use&nbsp;`exports.implementation`, the&nbsp;rest&nbsp;re‑assigns&nbsp;`module.exports`.

I&nbsp;prefer&nbsp;avoiding re‑assigning&nbsp;`module.exports` wherever&nbsp;possible and&nbsp;this&nbsp;was&nbsp;necessary before&nbsp;<https://github.com/jsdom/jsdom/commit/59b7dcf67f00aaa9a69dc81858aa3090265f3e6e>.

Reply:
Well arguably all intermediate parts of this PR are "broken" in some ways – and that's why they are not merged; it's just that clicking the "commit suggestion" button makes it more broken.

If you are so concerned about this, why don't you apply the change manually then?